company_website/static/main_page/testimonials.sass

```diff
@@ -0,0 +1,237 @@
+@import "partials/constants"
+
+$nav-button-width: 54px
+$testimonial-max-width: 1164px
+$testimonial-breakpoint: calc($nav-button-width * 2 + $testimonial-max-width + 1px)
+$nav-button-breakpoint: 1294px
+$testimonial-text-breakpoint: 847px
+$testimonial-text-breakpoint-2: 549px
+
+.background-image
+    position: absolute
+    z-index: -1
+
+.testimonials
+    font-family: $font-family
+    font-weight: 400
+    max-width: $max-section-width
+    margin: auto
+
+    @keyframes highlight
```

Comment:
Is it works on all browser? Because I know that there is possibilty to add `keyframes` with -webkit prefix etc. to adjust it to other browsers, but I my not sure

Reply:
This feature in particular works on all browsers I tested.